packages/default-catalog/api/catalog.ts

```diff
@@ -115,6 +115,7 @@ export default ({ config, db }) => async function (req, res, body) {
       auth: auth
     }, async (_err, _res, _resBody) => { // TODO: add caching layer to speed up SSR? How to invalidate products (checksum on the response BEFORE processing it)
       if (_err || _resBody.error) {
+        console.error(_err || _resBody.error)
```

Comment:
Use the [Logger.error](https://github.com/DivanteLtd/storefront-api/blob/master/packages/lib/logger.ts#L35) here and not console.error.

Reply:
done :)